helix-core/src/main/java/org/apache/helix/controller/rebalancer/util/DelayedRebalanceUtil.java

```diff
@@ -128,6 +129,14 @@ private static Set<String> getActiveNodes(Set<String> allNodes, Set<String> live
     return activeNodes;
   }
 
+  private static Set<String> filterEvacuatingInstances(Map<String, InstanceConfig> instanceConfigMap,
+      Set<String> liveEnabledNodes) {
```

Comment:
Would recommend naming variable something more generic than `liveEnabledNodes`(maybe just `nodes`) as we could use this method somewhere else in the class later to filter out evacuating nodes. We can't guarantee nor should we guarantee that this method is always used with live and enabled nodes.